src/blackmetal/config/blackmetal.yaml

```
@@ -0,0 +1,5 @@
+blackmetal:
+  ros__parameters:
+    bm_csIP: "192.168.1.3"
+    wheelRadius: 0.1
+    chasis: 0.5
```

Toto je asi zbytocna vrsta:  'ros__parameters'
```suggestion
blackmetal:
  bm_csIP: "192.168.1.3"
  wheelRadius: 0.1
  chasis: 0.5
```

---

V ros2 musi byt spraveny config file takto. Vpodstate sa da sparvit to aby sa dalo viacero node do jedneho suboru